lib/game.rb

```diff
@@ -0,0 +1,67 @@
+require_relative './score'
+
+class Game
+  attr_reader :throws, :score, :frame_count, :last_frame
+
+  MAX_FRAME = 10
+
+  def initialize(score = Score.new)
+    @throws = []
+    @frame_complete = true
```

Comment:
Why does @frame_complete start on true?

Reply:
There really is no reason! Will change that!